pkg/apiserver/handler_auth.go

```diff
@@ -14,17 +14,20 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package handlers
+package apiserver
```

Comment:
These handlers were moved so NewRequestAuthenticator and WithImpersonation can share the private method for setting the Authentication-Info header.